pkg/daemon/update_test.go

```diff
@@ -318,6 +318,57 @@ func TestUpdateSSHKeys(t *testing.T) {
 	}
 }
 
+// This test should fail until Ignition validation enabled.
+// Ignition validation does not permit writing files to relative paths.
+func TestInvalidIgnConfig(t *testing.T) {
+	// expectedError is the error we will use when expecting an error to return
+	expectedError := fmt.Errorf("broken")
```

Comment:
inline this error since it's not used anywhere else (also, golang prefers errors to be in the form of `expectedErr` but you can drop it altogether here)